docs/DeveloperGuide.md

```diff
@@ -784,6 +790,33 @@ Use case ends.
 2. User finds buyers that are no longer relevant (e.g. already bought a house).
 3. User executes delete command on these buyers.
 
+### Use case: Sort properties
+
+**MSS:**
+
+1. User chooses to sort properties by a specified field in a specified order.
+2. User executes sort properties command on the currently displayed properties list.
+
+Use case ends.
+
+### Use case: Sort buyers
+
+**MSS:**
+
+1. User chooses to sort buyers by a specified field in a specified order.
+2. User executes sort buyers command on the currently displayed buyer list.
+
+Use case ends.
+
+### Use case: List buyers
+
+**MSS:**
+
+1. User chooses to list buyers
+2. User executes list buyers command.
```

Comment:
3. Cobb shows a list of all the buyers currently saved?

Reply:
Oops I accidentally copied this from on top, but i'll help to change that!